physx/source/geomutils/src/gjk/GuGJKType.h

```diff
@@ -151,7 +151,7 @@ namespace Gu
 
 		virtual Ps::aos::Vec3V getCenter() const { return mAToB.transform(getConvex<Convex>().getCenter()); }
 
-		PX_FORCE_INLINE Ps::aos::PsMatTransformV& getRelativeTransform(){ return mAToB; }
+		PX_FORCE_INLINE const Ps::aos::PsMatTransformV& getRelativeTransform() const { return mAToB; }
```

Comment:
Is this the only clang-related change?

Reply:
Yes, this and the misleading indentation warning fixes were the only changes to the code that were necessary. Everything else was ignoring warnings in CMake or linking issues.